frontend/src/lib/stores/preferences.ts

```diff
@@ -14,6 +14,10 @@ currentGenres.subscribe(value => {
   }
 })
 
+export const isBurgerMenuOpen = writable<boolean>(
+  (browser && localStorage.isBurgerMenuOpen) || false
+)
+
```

Comment:
We don't need a special store for this, just use the normal one

Reply:
Done